lib/dradis/plugins/mapping.rb

```diff
@@ -0,0 +1,60 @@
+
+# When you call provides :upload in your Engine, this module gets included.
+module Dradis::Plugins::Mapping
+  extend ActiveSupport::Concern
+
+  included do
+    module_parent.extend ClassMethods
+  end
+
+  module ClassMethods
+
+    def component
+      meta[:name].to_s
+    end
+
+    def field_names(source:, destination: nil, field_type: 'destination')
```

Comment:
I understand the code, but under what conditions we want to get the list of mapped fields rather than the full list of fields? Seems weird!